release/src/main/scripts/run_rc_validation.sh

```diff
@@ -55,6 +55,18 @@ function clean_up(){
 
   rm -rf ${LOCAL_BEAM_DIR}
   echo "* Deleted workspace ${LOCAL_BEAM_DIR}"
+
+  if [[ -n `which gcloud` ]]; then
+    if [[ -n ${KAFKA_CLUSTER_NAME} ]]; then
+        echo "-----------------------Clean up Kafka Cluster on GKE------------------------"
```

Comment:
will this cleanup affect any running tests outside of RC validation?

Reply:
The variable KAFKA_CLUSTER_NAME is created below in L511. This clean up should only trigger when there is a kafka cluster being created there. Unless the shell already has an environment variable named "KAFKA_CLUSTER_NAME" and used for testing, and do not set `python_xlang_kafka_taxi_dataflow` to true in this test, it is expected not affecting running test.